config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.ttl

```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
+
+<http://data.lblod.info/id/bestuurseenheden/c8f33e80-6f19-4ba1-a758-9d85f42d28d4> <http://data.lblod.info/vocabularies/erediensten/betrokkenBestuur> <http://data.lblod.info/id/betrokkenLokaleBesturen/635237108DE5818A7C9F41D1> .
```

Comment:
Any reason why the betrokken itself is not added but only the link between the municipality and the betrokken ?

Reply:
Also there seem to be extra commits in the PR :grin:

Reply:
> Also there seem to be extra commits in the PR :grin:

Yes I've noticed that, but I don't know why they're showing here as in my commit history I've only committed the two `20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.*` files :confused: 

> Any reason why the betrokken itself is not added but only the link between the municipality and the betrokken ?

I'm not sure, we need this link ? I've never worked on CVP before.

Reply:
For the extra commit, I think it's because you based your branch on master (where those commits are) but you're merging to dev, so they are showing. It's not a big deal if the commit can also go to the dev branch

And CVP is a place where we gather all public data to be able to query it, so I suppose that we also expect the link to be there yes. When I check other triples of the same types and predicates, the link seems to be there

Reply:
Indeed, I'll check and add the missing link. Thanks for these insights :pray:

Reply:
I've made the changes, will test before pushing.

Reply:
yes I think it's expected as well :)